sale_procurement_group_by_line/model/sale.py

```diff
@@ -49,7 +49,13 @@ def action_ship_create(self, cr, uid, ids, context=None):
             groups = {}
 
             for line in order.order_line:
+                if order.state == 'shipping_except':
+                    group_id = getattr(line.procurement_group_id, 'id')
```

Comment:
This looks like a void statement to me. *group_id* is not used in the loop, and overwritten right after.